pages/boxscore/boxscore.cfm

```diff
@@ -14,20 +14,32 @@
 </cfquery>
 
 <cfquery name="getTeamsPlaying" datasource="roundleague">
-    SELECT scheduleID, WEEK, a.teamName AS Home, b.teamName AS Away, s.startTime, Date_FORMAT(s.date, "%M %d %Y") AS Date, s.homeScore, s.awayscore
+    SELECT scheduleID, WEEK, a.teamName AS Home, b.teamName AS Away, s.startTime, Date_FORMAT(s.date, "%M %d %Y") AS Date, s.homeScore, s.awayscore, a.teamID as HomeTeamID, b.teamID as AwayTeamID
     FROM schedule s
     LEFT JOIN teams as a ON s.hometeamID = a.teamID
     LEFT JOIN teams as b ON s.awayTeamID = b.teamID
     WHERE s.scheduleID = <cfqueryparam cfsqltype="CF_SQL_INTEGER" value="#url.scheduleID#">
     ORDER BY WEEK, startTime
 </cfquery>
 
+<cfquery name="getWinsAndLossesHomeTeam" datasource="roundleague">
+    SELECT Wins,Losses
+    FROM standings
+    WHERE teamID = #getTeamsPlaying.HomeTeamID# AND SeasonID = #session.CurrentSeasonID#
```

Comment:
```
    WHERE teamID = #getTeamsPlaying.HomeTeamID# AND SeasonID = #session.CurrentSeasonID#
```

When we have coldfusion variables (in this case #getTeamsPlaying.HomeTeamID#) within a cfquery we always want to wrap that variable in a `<cfqueryparam>` so it would look like this

```suggestion
    WHERE teamID = <cfqueryparam cfsqltype="CF_SQL_INTEGER" value="#getTeamsPlaying.HomeTeamID#">
    AND SeasonID = <cfqueryparam cfsqltype="CF_SQL_INTEGER" value="#session.CurrentSeasonID#">
```

Let's also add AND to the next line any time we have a where clause. Everything else looks good tho